boards/nrf51dk/README.md

```diff
@@ -8,24 +8,48 @@ All code for the kit is compatible with the nRF51822 as well.
 
 ## Necessary tools
 
-Programming the nRF51 DK requires `JLinkExe`, a JTAG programming application.
+There are two ways to program the nRF51DK: JTAG or the mbed file system. If
+you choose to use JTAG (the recommended approach), this requires `JLinkExe`, 
+a JTAG programming application.
 It can be downloaded from [Segger](https://www.segger.com/downloads/jlink),
 you want the "Software and Documentation Pack".
 
 
 ## Programming the kernel
 
-To program the nRF51 DK, a Segger JTAG chip is included on the board. A USB
+### Programming with JTAG (recommended)
+
+The nRF51DK, a Segger JTAG chip is included on the board. A USB
 connection to a computer allows code to be uploaded to the board.
 
 To program the Tock kernel onto the nRF51 DK, run:
 
 ```bash
-$ make TOCK_PLATFORM=nrf_pca10001 program
+$ make TOCK_BOARD=nrf51dk program
+```
+
+This will build `boards/nrf51dk/target/nrf51/release/nrf51dk` and 
+program it on the board using `JLinkExe`.
+
+### Programming with mbed file system
```

Comment:
Why do we insist on explaining mbed? It's so different from our other (more canonical platform), it's more cumbersome, and if someone really cares they can of course do it.